src/log.cpp

```diff
@@ -80,8 +91,24 @@ void log_to_file(ru_severity min_sev, const char* file_path) {
   get_single_logger()->log_to_file(min_sev, file_path);
 }
 
+#ifdef RS2_USE_ANDROID_BACKEND
+void android_log_msg(ru_severity sev, const std::string& msg) {
```

Comment:
indent this line

Reply:
Depending on who you ask, code inside preprocessor ifs shouldn't be indented.

The branch below (in the `log_msg()` call) is not indented so I've not indented this either. I have however fixed the indentation for the function itself in a53ac59c4

Reply:
Great, it was either indent this or unindent the rest :)